internal/criocli/criocli.go

```diff
@@ -1092,7 +1092,7 @@ func getCrioFlags(defConf *libconfig.Config) []cli.Flag {
 		&cli.StringFlag{
 			Name:      "version-file-persist",
 			Usage:     "Location for CRI-O to lay down the persistent version file. It is used to check if crio wipe should wipe images, which should only happen when CRI-O has been upgraded.",
-			Value:     defConf.VersionFile,
+			Value:     "",
```

Comment:
I think this can stay defConf.VersionFilePersist, as it will future proof it. I think the documentation changes will remain

Reply:
done